libs/challenge-registry/user-profile/src/lib/user-profile.component.html

```diff
@@ -1,27 +1,60 @@
-<main>
-  <div class="container" *ngIf="user$ | async as user">
-    <challenge-registry-user-profile-bar [user]="user" [numOrgs]="orgs.length">
-    </challenge-registry-user-profile-bar>
+<div class="base-container" *ngIf="user$ | async as user">
+  <div class="base-container1">
+    <section id="top" class="base-top-section">
+      <challenge-registry-avatar
+        class="base-profile-pic"
+        [avatar]="userAvatar"
+      ></challenge-registry-avatar>
+      <span class="base-text">Awesome User</span>
+      <mat-icon aria-hidden="true" class="base-verified">verified_outline</mat-icon>
+      <span class="base-text01">@awesome-user</span>
+      <div class="base-usertypetag">
+        <span class="base-text02">User</span>
+      </div>
+      <div class="base-createdat">
+        <span class="base-text03">ROCC user since</span>
+        <span class="base-text04">01/01/2020</span>
+      </div>
+      <div class="base-updatedat">
+        <span class="base-text05">Profile last updated</span>
+        <span class="base-text06">03/31/2022</span>
+      </div>
+      <challenge-registry-user-profile-stats
+        class="base-basic-stats"
+        [loggedIn]="loggedIn"
```

Comment:
- if not loggin, send email the user.
- if loggin and it's different user, send email the user.
- if loggin and it's the same user, enable to edit

Reply:
#538